docs/_templates/application.njk

```diff
@@ -104,37 +104,37 @@
             {% endblock %}
           </header>
 
-          {% block content %}
-            {# page content goes here #}
-          {% endblock %}
-        {% endblock %}
-
-        {% if page.also %}
-          {% include 'partials/see-also.njk' %}
-        {% endif %}
-      </main>
-
-      {% if hasSiderail %}
-        <aside class="hxSiderail">
-          {% block siderail %}
-            <header>
-              <h1 class="hxHeading-4">On this page:</h1>
-            </header>
-
-            <nav class="article-toc">
-              {{ page.raw | toc | safe }}
-              {% block toc %}
-                {# extra links go here #}
+          <div>
+            {% block contentWrapper %}
+              {% block content %}
+                {# page content goes here #}
               {% endblock %}
+
               {% if page.also %}
-                <a href="#see-also">See Also</a>
+                {% include 'partials/see-also.njk' %}
               {% endif %}
-              <hr class="hxDivider" />
-              <a class="top-link" href="#top">Back to Top</a>
-            </nav>
-          {% endblock %}
-        </aside>
-      {% endif %}
+            {% endblock %}
+          </div>
+
+          {% if hasSiderail %}
+            <aside class="hxSiderail">
+              {% block siderail %}
+                <nav class="toc" aria-label="table of contents">
+                  {{ page.raw | toc | safe }}
+                  {% block toc %}
+                    {# extra links go here #}
+                  {% endblock %}
+                  {% if page.also %}
+                    <a href="#see-also">See Also</a>
+                  {% endif %}
+                  <hr class="hxDivider" />
+                  <a class="top-link" href="#top">Back to Top</a>
+                </nav>
+              {% endblock %}
+            </aside>
+          {% endif %}
+        {% endblock %}
+      </main>
```

Comment:
`<header>`, "body" (`<div>`), and siderail (`<aside class="hxSiderail">`) all now contained within `{% block main %}`